core/src/main/java/io/confluent/kafka/schemaregistry/rest/resources/ConfigResource.java

```diff
@@ -112,11 +112,15 @@ public Config getSubjectLevelConfig(@PathParam("subject") String subject) {
     Config config = null;
     try {
       AvroCompatibilityLevel compatibilityLevel = schemaRegistry.getCompatibilityLevel(subject);
+      if (compatibilityLevel == null) {
+          throw Errors.subjectNotFoundException();
+      }
       config = new Config(compatibilityLevel == null ? null : compatibilityLevel.name);
```

Comment:
Given the null check above, we can just use 
`config = new Config(compatibilityLevel.name);`